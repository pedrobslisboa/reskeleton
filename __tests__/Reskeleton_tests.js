'use strict';

var Jest = require("@glennsl/bs-jest/src/jest.js");
var React = require("react");
var Reskeleton = require("../src/Reskeleton.js");
var ReactTestingLibrary = require("bs-react-testing-library/src/ReactTestingLibrary.bs.js");

Jest.describe("getByText", (function (param) {
        Jest.test("should render with className", (function (param) {
                return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.render(undefined, undefined, undefined, undefined, undefined, React.createElement(Reskeleton.make, {
                                        className: "test-class-name"
                                      })).container));
              }));
        Jest.test("should render", (function (param) {
                return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.render(undefined, undefined, undefined, undefined, undefined, React.createElement(Reskeleton.make, {
                                        variant: /* block */888960333
                                      })).container));
              }));
        Jest.test("should render as text", (function (param) {
                return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.render(undefined, undefined, undefined, undefined, undefined, React.createElement(Reskeleton.make, {
                                        variant: /* text */-856044371
                                      })).container));
              }));
        return Jest.test("should render as text", (function (param) {
                      return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.render(undefined, undefined, undefined, undefined, undefined, React.createElement(Reskeleton.make, {
                                              variant: /* text */-856044371,
                                              count: 4
                                            })).container));
                    }));
      }));

/*  Not a pure module */
