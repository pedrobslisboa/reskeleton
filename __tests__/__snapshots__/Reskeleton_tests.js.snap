// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getByText should render 1`] = `
<div>
  <div
    class=" css-ykgghw"
    style="height: 100%; width: 100%; border-radius: 4px;"
  >
    <div />
  </div>
</div>
`;

exports[`getByText should render as text 1`] = `
<div>
  <span
    class=" css-ykgghw css-11gtyes"
    style="width: 100%;"
  >
    <span />
  </span>
</div>
`;

exports[`getByText should render as text 2`] = `
<div>
  <span
    class=" css-ykgghw css-11gtyes"
    style="width: 100%;"
  >
    <span />
  </span>
  <span
    class=" css-ykgghw css-11gtyes"
    style="width: 100%;"
  >
    <span />
  </span>
  <span
    class=" css-ykgghw css-11gtyes"
    style="width: 100%;"
  >
    <span />
  </span>
  <span
    class=" css-ykgghw css-11gtyes"
    style="width: 100%;"
  >
    <span />
  </span>
</div>
`;

exports[`getByText should render with className 1`] = `
<div>
  <div
    class=" css-ykgghw test-class-name"
    style="height: 100%; width: 100%; border-radius: 4px;"
  >
    <div />
  </div>
</div>
`;
